opennlp-tools/src/main/java/opennlp/tools/util/HashList.java

```diff
@@ -27,27 +26,46 @@
  * Class which creates mapping between keys and a list of values.
  */
 @SuppressWarnings("unchecked")
-public class HashList extends HashMap {
+public class HashList extends HashMap<Object, Object> {
```

Comment:
If you do that the overridden `put()` can't return a list. `HashMap.put()` returns an object of the type that's being added (`V`) but the `put()` here returns the list of objects. So that's why I stuck with `<Object, Object>` to remove the raw types.